docs/source/api/v4/cdns_id_queue_update.rst

```diff
@@ -70,9 +80,10 @@ Response Structure
 	Whole-Content-Sha512: rBpFfrrP+9IFkwsRloEM+v+I8MuBZDXqFu+WUTGtRGypnAn2gHooPoNQRyVvJGjyIQrLXLvqjEtve+lH2Tj4uw==
 	X-Server-Name: traffic_ops_golang/
 	Date: Wed, 14 Nov 2018 21:02:07 GMT
-	Content-Length: 41
+	Content-Length: 54
 
 	{ "response": {
 		"action": "queue",
-		"cdnId": 2
+		"cdnId": 2,
+		"typeID": 11
```

Comment:
Since the response is staying the same, this example response change should be undone as well.